schematic_kicad/Transistor_FET.dcm

```diff
@@ -1404,6 +1422,18 @@ K Dual N-Channel P-Channel MOSFET
 F https://www.fairchildsemi.com/datasheets/FD/FDS9934C.pdf
 $ENDCMP
 #
+$CMP IPD50R380CE
```

Comment:
I'm not sure what happend but i'm guessing these changes shouldn't be here

Reply:
It seems like things got moved around and copied unintentionally, I manually removed the copies and moved some things it should be good now, the only thing that is relevant is IRFR010.